features/configuration/fail_if_no_examples.feature

```diff
@@ -24,8 +24,8 @@ Feature: fail if no examples
 
   Scenario: Examples file name is not matched by RSpec pattern, thus there are no examples run
     When I run `rspec`
-    Then it should fail with "0 examples, 0 failures"
+    Then it should fail with output "0 examples, 0 failures"
```

Comment:
Given a step change is required to make this happen, is it necessary?

Reply:
I don't think so, no.